composer.json

```diff
@@ -139,7 +146,11 @@
             "GPBMetadata\\Google\\": "CommonProtos/metadata",
             "GPBMetadata\\Google\\Analytics\\Admin\\": "AnalyticsAdmin/metadata",
             "GPBMetadata\\Google\\Analytics\\Data\\": "AnalyticsData/metadata",
+            "GPBMetadata\\Google\\Appengine\\": "AppEngineAdmin/metadata",
```

Comment:
I just wanted to call out the inconsistency in case it was accidental.

Reply:
Good eye, thank you! It is intentional and unfortunately part of how protobuf generates metadata classes.